contact.html

```diff
@@ -1,105 +1,119 @@
+  <nav id="nav-contact" class="js--nav">
+
+    <div class="row main-nav js--main-nav">
+      <div class="desktop-width-limit">
+        <a class="mobile-nav-icon js--mobile-nav-icon">
+          <div id="js--hamburger" class="exist">
+            <svg width="18px" height="12px" viewBox="0 0 18 12" version="1.1"
+                 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+              <!-- Generator: sketchtool 49 (51002) - http://www.bohemiancoding.com/sketch -->
+              <title>0E750DAF-0596-42E2-94B9-360ED2184522</title>
+              <desc>Created with sketchtool.</desc>
+              <defs></defs>
+              <g id="MOBILE" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"
+                 stroke-linecap="round">
+                <g id="Mobile-main" transform="translate(-19.000000, -26.000000)" stroke="#2F4858">
+                  <g id="Intro" transform="translate(-71.000000, 0.000000)">
+                    <g id="mobile/hamburger" transform="translate(87.000000, 20.000000)">
+                      <g id="hamburger" transform="translate(4.000000, 6.000000)">
+                        <path d="M0,1 L16,1" id="Line" stroke-width="2"></path>
+                        <path d="M0,6 L16,6" id="Line-Copy" stroke-width="2"></path>
+                        <path d="M0,11 L16,11" id="Line-Copy-2" stroke-width="2"></path>
+                      </g>
+                    </g>
+                  </g>
+                </g>
+              </g>
+            </svg>
+          </div>
+          <div id="js--menu-go-back-arrow" class="not-exist">
+            <svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1"
+                 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+              <!-- Generator: sketchtool 49 (51002) - http://www.bohemiancoding.com/sketch -->
+              <title>88E30192-77DD-4CFC-AA0C-561F9344DC7E</title>
+              <desc>Created with sketchtool.</desc>
+              <defs></defs>
+              <g id="MOBILE" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"
+                 stroke-linecap="round">
+                <g id="Menu" transform="translate(-20.000000, -24.000000)" stroke="#2F4858">
+                  <g id="mobile/back" transform="translate(16.000000, 20.000000)">
+                    <g id="Group" transform="translate(5.000000, 5.000000)">
+                      <polyline id="Triangle" stroke-width="2" stroke-linejoin="round"
+                                points="7 0 0 7 7 14"></polyline>
+                      <path d="M1,7 L14,7" id="Line-Copy" stroke-width="2"></path>
+                    </g>
+                  </g>
+                </g>
+              </g>
+            </svg>
+          </div>
+        </a>
+
+        <div class="logo-space">
+          <a class="js--black exist mobile-logo animated-logo-hover" href="index.html"
+             id="black-logo">
             <!--
-word n
```

Comment:
When formatting the logo section mind the lack of the space in HTML between the <svg> tags. No gap left between SVGs is on purpose. This is very strange but you need it so ther is no additional space between svg files.